addons/mail/static/src/components/discuss/discuss.scss

```diff
@@ -1,96 +1,7 @@
-// ------------------------------------------------------------------
-// Layout
-// ------------------------------------------------------------------
-
 .o-autogrow {
     flex: 1 1 auto;
```

Comment:
The default `flex` value is `0 1 auto` which means we could replace this class with `flex-grow-1`.
But it seems that `o-autogrow` is used in many components, so I’m not sure if we should make this change in this PR or in a new one.

What do you think @alexkuhn ?

Reply:
Better make a single PR that removes `o-autogrow` in this file, and replace all occurrences of `o-autogrow` in templates by `flex-grow-1`. Thanks!

Reply:
Ok 👍 

@karlosvalverde Can you handle it?